grails-app/services/org/broadinstitute/orsp/IssueService.groovy

```diff
@@ -237,6 +237,13 @@ class IssueService {
         newIssue
     }
 
+    void deleteIssue(String projectKey) {
+        Issue issue = queryService.findByKey(projectKey)
+        if(issue != null) {
+            issue.delete(flush: true)
```

Comment:
Question: Have you tested that this deletes associations (attachments, submissions, extra properties) as well? I've never been very confident of the framework-delegated functions like this.